betterconf/providers.py

```diff
@@ -27,18 +28,20 @@ class INIProvider(AbstractProvider):
     def __init__(
         self,
         files: Union[str, List[str]],
-        delimiters: Tuple[str] = ("=", ":"),
-        comment_prefixes: Tuple[str] = ("#", ";"),
+        delimiters: Sequence[str] = ("=", ":"),
+        comment_prefixes: Sequence[str] = ("#", ";"),
```

Comment:
It's kinda not the same. I think `Union[Tuple[str, ...], List[str]]` will be better here, because:

- user should be able to have longer than 1 symbol delimiters or comments
- we don't know all possible `.ini` format configurations